JavaImportTool/src/javaImportTool/ASTConverter.java

```diff
@@ -654,17 +653,19 @@ public void processTypeParameters(List<TypeParameter> params) throws ConversionE
 		}
 	}
 	
-	public void processParameters(List<SingleVariableDeclaration> params) throws ConversionException
+	public void processParameters(List<VariableDeclaration> params) throws ConversionException
 	{
 		int i = 0;
-		for(SingleVariableDeclaration arg : params)
+		for(VariableDeclaration arg : params)
 		{
 			Node a = containers.peek().child("arguments").add(new Node(null,"FormalArgument", i++));
 			a.setSymbol(arg.getName().getIdentifier());
 			setModifiersAndAnnotations(a, Modifier.PUBLIC, null);
 			
-			Node type = typeExpression(arg.getType(), "typeExpression");
-			a.setChild("typeExpression", addExtraDimensions(type,arg.getExtraDimensions()));
+			if (arg instanceof SingleVariableDeclaration) {
```

Comment:
Well, the doc says that in the fragment case, there is either no type (lambda) or the type is part of the parent (I guess in cases like `public int a, b` the b is a fragment.

The ideal thing is to distinguish those two cases and do nothing in the first but actually put the proper type in the second.